pkg/cspc/algorithm/build_deploy.go

```diff
@@ -255,6 +255,14 @@ func getMayaExporterImage() string {
 	return image
 }
 
+func getServiceAccountName() string {
```

Comment:
Can we move it to some common place? So that will be making use by all the operators that exist in this repo?